plugins/source/shopify/client/client.go

```diff
@@ -72,6 +75,9 @@ func getServiceClient(logger zerolog.Logger, spec *Spec) (*shopify.Client, error
 	if spec.MaxRetries < 1 {
 		spec.MaxRetries = 30
 	}
+	if spec.PageSize < 1 {
```

Comment:
What is the benefit of having page size be part of the spec, rather than always choosing the largest possible page size?

Reply:
The benefit is actually documented in this PR:
> - `page_size` (integer, optional. Default: 50):
>  Maximum number of items queried each request. Find an optimum value to balance amount of data fetched and requests timing out. Maximum value 250.

Large page sizes tend to time out if you have a large account.

Reply:
Ah okay, gotcha 👍